EKF/estimator_interface.h

```diff
@@ -394,6 +394,26 @@ class EstimatorInterface
 	warning_event_status_u _warning_events{};
 	information_event_status_u _information_events{};
 
+	uint64_t _baro_sample_delayed_time_us_last{0};
+	uint32_t _baro_hgt_counter{0};
+	AlphaFilter<float> _baro_hgt_delayed{0.1f};
```

Comment:
```suggestion
	AlphaFilter<float> _baro_hgt_delayed_lpf{0.1f};
```
I would like to have the `_lpf` prefix to make it obvious that this is not just holding "baro_sample_delayed"

Reply:
same for the other ones